index.html

```diff
@@ -532,6 +532,19 @@ <h3 id="navigation-scope-security-considerations">
           security reasons. It ensures that users are always aware of which
           <a>origin</a> they are interacting with.
         </p>
+        <p>
+          Despite this, there is still a potential spoofing risk, if an
+          installed app pretends to navigate to an out-of-scope site on another
+          <a>origin</a>. The site shows a fake version of the user agent's
+          prominent out-of-scope UI, indicating to the user that it is on
+          another origin. However, in reality, the user has never navigated
+          away from the installed app's origin, and the user agent is not
+          showing any out-of-scope UI. User agents MAY wish to ensure that the
```

Comment:
We need to drop the MAY here, as it's impossible we can ever detect this (as stated below).

Reply:
To be clear, I think all we can do is really just point out that this can happen, but there is no much we can do about it.